components/Footer.js

```diff
@@ -35,170 +37,386 @@ const SocialLink = styled.a`
 `;
 
 const MenuLink = styled(StyledLink)`
-  color: #6e747a;
+  color: #4e5052;
   display: block;
-  fontsize: 1.4rem;
-  fontweight: 400;
+  font-size: 14px;
+  line-height: 17px;
+  font-weight: 500;
+  letter-spacing: -0.1px;
   margin: 0;
   padding: 0;
 `;
 
 const FlexList = styled.ul([], ...Box.componentStyle.rules, ...Flex.componentStyle.rules);
 
+const messages = defineMessages({
+  platform: {
+    id: 'platform',
+    defaultMessage: 'Platform',
+  },
+  'platform.explainerVideo': {
+    id: 'platform.explainerVideo',
+    defaultMessage: 'Explainer video',
+  },
+  'platform.howItWorks': {
+    id: 'platform.howItWorks',
+    defaultMessage: 'How it works',
+  },
+  'platform.useCases': {
+    id: 'platform.useCases',
+    defaultMessage: 'Use Cases',
+  },
+  'platform.signup': {
+    id: 'platform.signup',
+    defaultMessage: 'Sign up',
+  },
+  'platform.login': {
+    id: 'platform.login',
+    defaultMessage: 'Login',
+  },
+  join: {
+    id: 'join',
+    defaultMessage: 'Join',
+  },
+  'join.createACollective': {
+    id: 'join.createACollective',
+    defaultMessage: 'Create a Collective',
+  },
+  'join.discover': {
+    id: 'join.discover',
+    defaultMessage: 'Discover',
+  },
+  'join.findAFiscalHost': {
+    id: 'join.findAFiscalHost',
+    defaultMessage: 'Find a fiscal host',
+  },
+  community: {
+    id: 'community',
+    defaultMessage: 'Community',
+  },
+  'community.openSource': {
+    id: 'community.openSource',
+    defaultMessage: 'Open Source',
+  },
+  'community.slack': {
+    id: 'community.slack',
+    defaultMessage: 'Slack',
+  },
```

Comment:
We don't need to translate this one, Slack is always Slack no matter what the language is. A good alternative is to remove it from there, and do something like:

```jsx
{messages[key] ? intl.formatMessage(messages[key]) : key}
```